### DISCUSSION TITLE
`自定义顶栏`的`新标签页打开`选项未开启时应在当前标签页打开搜索栏的搜索结果

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#5020] [功能建议 / Ideas]
`自定义顶栏`的`新标签页打开`选项未开启时应在当前标签页打开通过`自定义顶栏`的搜索栏打开的搜索页面。

**Comment by the1812** [comment]
`新标签页打开` 的描述是 `控制顶栏内链接是否在新标签页打开`. 搜索我觉得是一个动作, 不是链接

**Reply by Geeyun-JY3** [reply]
那是否考虑给个选项呢，隔壁 [BewlyBewly](https://github.com/BewlyBewly/BewlyBewly) 就有给：
![image](https://github.com/user-attachments/assets/56cee745-d611-4e17-8348-620737250b1c)

**Reply by the1812** [reply]
可以做, 但优先级不高(
